### PR TITLE
[zh]Sync cluster-intro.html、configure-java-microservice-interactive.html、update-interactive.html

### PR DESCRIPTION
**Pull request by yyyhhhh**
[zh]Sync /tutorials/kubernetes-basics/create-cluster/cluster-intro.html
[zh]Sync /configuration/configure-java-microservice/configure-java-microservice-interactive.html
[zh]Sync /tutorials/kubernetes-basics/update/update-interactive.html

### CODE REVIEW

File: content/zh-cn/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.html [hunk 1]
```diff
@@ -126,14 +126,14 @@ <h2 style="color: #3771e3;">集群图</h2>
...
                 -->
                 <p><b>节点是一个虚拟机或者物理机，它在 Kubernetes 集群中充当工作机器的角色。</b>
-                每个节点都有 Kubelet，它管理节点而且是节点与控制面通信的代理。
-                节点还应该具有用于​​处理容器操作的工具，例如 Docker 或 rkt。
-                处理生产级流量的 Kubernetes 集群至少应具有三个节点，因为如果一个节点出现故障其对应的
-                <a href="/zh-cn/docs/concepts/overview/components/#etcd">etcd</a> 成员和控制面实例都会丢失，
-                并且冗余会受到影响。你可以通过添加更多控制面节点来降低这种风险。</p>
+                    每个节点都有 Kubelet，它管理节点而且是节点与控制面通信的代理。
```
Comment (windsonsea) [inline content/zh-cn/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.html:132]: 为什么这里要缩进，可以 follow 原中文的 style 吗？

File: content/zh-cn/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.html [hunk 2]
```diff
@@ -149,13 +149,12 @@ <h2 style="color: #3771e3;">集群图</h2>
...
+                <p>When you deploy applications on Kubernetes, you tell the control plane to start the application containers. The control plane schedules the containers to run on the cluster's nodes. <b>Node-level components, such as the kubelet, communicate with the control plane using the <a href="/docs/concepts/overview/kubernetes-api/">Kubernetes API</a></b>, which the control plane exposes. End users can also use the Kubernetes API directly to interact with the cluster.</p>    
                 -->
                 <p>在 Kubernetes 上部署应用时，你告诉控制面启动应用容器。
-                控制面就编排容器在集群的节点上运行。
-                <b>节点使用控制面暴露的 <a href="/zh-cn/docs/concepts/overview/kubernetes-api/">Kubernetes API</a>
-                与控制面通信。</b>终端用户也可以使用 Kubernetes API 与集群交互。</p>
-
+                    控制面就编排容器在集群的节点上运行。
```
Comment (windsonsea) [inline content/zh-cn/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.html:155]: 同上，无需修改缩进

**Review by windsonsea — COMMENTED** [review]
Please provide a simple, clear, and meaningful title for your PR.

You have changed 3 files, but only mentioned 2 in the PR's description.

File: content/zh-cn/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.html (hunk 2 above)
Comment (yyyhhhh) [reply content/zh-cn/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.html]: 谢谢大佬指点

File: content/zh-cn/docs/tutorials/configuration/configure-java-microservice/configure-java-microservice-interactive.html [hunk 3]
```diff
@@ -15,23 +20,31 @@
...
+                </h2>
+                <p>
+                  <!--The interactive tutorial for creating a cluster is not available.-->
+                  创建集群的交互式教程不可用。
+                  <!--For more information, see the
+                  <a href="https://kubernetes.io/blog/2023/02/14/kubernetes-katacoda-tutorials-stop-from-2023-03-31/">
+                    shutdown announcement</a>.-->
+                  有关更多信息，请参阅
+                  <a>"https://kubernetes.io/blog/2023/02/14/kubernetes-katacoda-tutorials-stop-from-2023-03-31/"
+                  关闭公告</a>
```
Comment (windsonsea) [inline content/zh-cn/docs/tutorials/configuration/configure-java-microservice/configure-java-microservice-interactive.html:41]:
```suggestion
                  关闭公告</a>。
```

File: content/zh-cn/docs/tutorials/configuration/configure-java-microservice/configure-java-microservice-interactive.html [hunk 4]
```diff
@@ -4,8 +4,13 @@
 ---
 <!-- 
 ---
-title: "Interactive Tutorial - Configuring a Java Microservice"
+title: "Tutorial - Configuring a Java Microservice"
```
Comment (windsonsea) [inline content/zh-cn/docs/tutorials/configuration/configure-java-microservice/configure-java-microservice-interactive.html:7]: 对应的中文 front matter 也要修改吧

File: content/zh-cn/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.html [hunk 5]
```diff
@@ -149,13 +149,12 @@ <h2 style="color: #3771e3;">集群图</h2>
...
+                <p>When you deploy applications on Kubernetes, you tell the control plane to start the application containers. The control plane schedules the containers to run on the cluster's nodes. <b>Node-level components, such as the kubelet, communicate with the control plane using the <a href="/docs/concepts/overview/kubernetes-api/">Kubernetes API</a></b>, which the control plane exposes. End users can also use the Kubernetes API directly to interact with the cluster.</p>    
                 -->
                 <p>在 Kubernetes 上部署应用时，你告诉控制面启动应用容器。
                 控制面就编排容器在集群的节点上运行。
-                <b>节点使用控制面暴露的 <a href="/zh-cn/docs/concepts/overview/kubernetes-api/">Kubernetes API</a>
+                <b>节点级组件,如 kubelet, 节点使用控制面暴露的 <a href="/zh-cn/docs/concepts/overview/kubernetes-api/">Kubernetes API</a>
```
Comment (windsonsea) [inline content/zh-cn/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.html:156]:
```suggestion
                <b>kubelet 这类节点级组件使用控制面暴露的 <a href="/zh-cn/docs/concepts/overview/kubernetes-api/">Kubernetes API</a>
```

File: content/zh-cn/docs/tutorials/kubernetes-basics/update/update-interactive.html [hunk 6]
```diff
@@ -6,38 +6,42 @@
 ---
 title: Interactive Tutorial - Updating Your App
 weight: 20
+headless: true
```
Comment (windsonsea) [inline content/zh-cn/docs/tutorials/kubernetes-basics/update/update-interactive.html:9]: 对应的中文 front matter 也要修改

File: content/zh-cn/docs/tutorials/kubernetes-basics/update/update-interactive.html [hunk 7]
```diff
@@ -6,38 +6,42 @@
...
+            </h2>
+            <p>
+              <!--The interactive tutorial for updating an application in your cluster is not available.
+              For more information, see the
+              <a href="https://kubernetes.io/blog/2023/02/14/kubernetes-katacoda-tutorials-stop-from-2023-03-31/">
+                shutdown announcement</a>.-->
+              更新集群中应用程序的交互式教程不可用。
+              有关更多信息，请参阅
+              <a href="https://kubernetes.io/blog/2023/02/14/kubernetes-katacoda-tutorials-stop-from-2023-03-31/">
+              关闭公告</a>.
```
Comment (windsonsea) [inline content/zh-cn/docs/tutorials/kubernetes-basics/update/update-interactive.html:39]:
```suggestion
              关闭公告</a>。
```

File: content/zh-cn/docs/tutorials/kubernetes-basics/update/update-interactive.html [hunk 8]
```diff
@@ -6,38 +6,42 @@
...
+              内容不可用
+            </h2>
+            <p>
+              <!--The interactive tutorial for updating an application in your cluster is not available.
+              For more information, see the
+              <a href="https://kubernetes.io/blog/2023/02/14/kubernetes-katacoda-tutorials-stop-from-2023-03-31/">
+                shutdown announcement</a>.-->
+              更新集群中应用程序的交互式教程不可用。
+              有关更多信息，请参阅
+              <a href="https://kubernetes.io/blog/2023/02/14/kubernetes-katacoda-tutorials-stop-from-2023-03-31/">
```
Comment (windsonsea) [inline content/zh-cn/docs/tutorials/kubernetes-basics/update/update-interactive.html:38]:
```suggestion
              <a href="https://kubernetes.io/zh-cn/blog/2023/02/14/kubernetes-katacoda-tutorials-stop-from-2023-03-31/">
```

File: content/zh-cn/docs/tutorials/configuration/configure-java-microservice/configure-java-microservice-interactive.html [hunk 9]
```diff
@@ -15,23 +20,31 @@
...
+                  内容不可用
+                </h2>
+                <p>
+                  <!--The interactive tutorial for creating a cluster is not available.-->
+                  创建集群的交互式教程不可用。
+                  <!--For more information, see the
+                  <a href="https://kubernetes.io/blog/2023/02/14/kubernetes-katacoda-tutorials-stop-from-2023-03-31/">
+                    shutdown announcement</a>.-->
+                  有关更多信息，请参阅
+                  <a>"https://kubernetes.io/blog/2023/02/14/kubernetes-katacoda-tutorials-stop-from-2023-03-31/"
```
Comment (windsonsea) [inline content/zh-cn/docs/tutorials/configuration/configure-java-microservice/configure-java-microservice-interactive.html:40]:
```suggestion
                  <a>"https://kubernetes.io/zh-cn/blog/2023/02/14/kubernetes-katacoda-tutorials-stop-from-2023-03-31/"
```

**Review by windsonsea — COMMENTED** [review]
这是 push 错了吗？怎么修改了 6 个文件，跟你 PR 标题、描述不符。

**Comment by 1000Delta** [comment]
Hi @yyyhhhh , thanks for your contribution, but your commit has wrong changes so it broked the build, you can check the detail in:

https://app.netlify.com/sites/kubernetes-io-main-staging/deploys/65659b60a7c262000869c6b1#L133

If you fix it and the build succeed, you can see the preview of your updates.

---

And you should change the title of your PR to match changes.

**Pull request closed without merging**